.clj-kondo/taoensso/encore/taoensso/encore.clj

```diff
@@ -0,0 +1,16 @@
+(ns taoensso.encore
```

Comment:
These changes should be removed, @siddarthkay ?

Reply:
removed from this PR till we fix : https://github.com/status-im/status-mobile/issues/17947